kubernetes/mlflow-tracking-operator-master/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM docker.io/centos/python-36-centos7
+
+USER root
+
+# When building pass `docker build --build-arg MLFLOW_VERSION=0.8.0`
+ARG mlflow_version
+
+LABEL architecture="x86_64"  \
+      com.redhat.component="opendatahub-mlflow-tracking-server"  \
+      maintainer="Zak Hassan <zak.hassan@redhat.com>" \
+      mlflow="$mlflow_version"
+
+RUN  pip install mlflow==$mlflow_version && pip install awscli
+COPY entrypoint.sh /etc/entrypoint.sh
```

Comment:
Do you need sudo to run entrypoint.sh? If so then I'd advise looking into ways you run this without root privileges because many kubernetes deployments do not grant users root access in containers for security purposes. These security features are on by default in Red Hat OpenShift, but not in Kubernetes.